src/objects/task1.js

```diff
@@ -0,0 +1,9 @@
+let human = {
+    firstName: "Jonas ",
+    lastName: "Jonaitis ",
+    age: " 30 ",
```

Comment:
Object field hasn't be a string, you can assign any data type, e.g. number, another object, array, function, etc. In your case - age should be a number